src/sentry/migrations/0516_migrate_world_map_widgets.py

```diff
@@ -0,0 +1,62 @@
+# Generated by Django 3.2.20 on 2023-07-18 21:10
+
+from django.db import migrations
+
+from sentry.models import DashboardWidgetQuery
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+def migrate_worldmap_widgets_to_table_widgets(apps, schema_editor):
+    # World map visulization has display_type 5
+    widgetQueries = DashboardWidgetQuery.objects.select_related("widget").filter(
+        widget__display_type=5
+    )
+
+    for widgetQuery in widgetQueries:
+        # Change widget display type from world_map:5 to table:4
+        widgetQuery.widget.display_type = 4
+
+        # Ensure condition has countr_code check
+        if "has:geo.country_code" not in widgetQuery.conditions:
+            widgetQuery.conditions = widgetQuery.conditions + " has:geo.country_code"
+
+        # Add region and country_code columns
+        if "geo.region" not in widgetQuery.columns:
+            widgetQuery.columns.insert(0, "geo.region")
+        if "geo.country_code" not in widgetQuery.columns:
+            widgetQuery.columns.insert(0, "geo.country_code")
```

Comment:
Why do we insert to the start of the list here? Is the order important?

Reply:
Yeah we care about the order here. We want country_code and region to be the first two columns/fields.